demo/index.html

```diff
@@ -126,9 +129,12 @@
       output.innerHTML = JSON.stringify(event.detail);
     }
 
-    function clickHandler(event) {
+    function submitHandler(event) {
       Polymer.dom(event).localTarget.parentElement.submit();
     }
+    function resetHandler(event) {
+      Polymer.dom(event).localTarget.parentElement.reset();
```

Comment:
In general? Yeah, but this is just for the demo...so it's just for this particular button that isn't moving around. Is this really needed? :(